src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Protocol/DelegatedTypes.cs

```diff
@@ -44,6 +44,6 @@ internal record DelegatedCompletionResolutionContext(
     object? OriginalCompletionListData);
 
 internal record DelegatedCompletionItemResolveParams(
-    TextDocumentIdentifier HostDocument,
+    VersionedTextDocumentIdentifier HostDocument,
```

Comment:
In order to properly use the TextDocumentSynchronizer we must have the HostDocumentVersion.